scripts/start-ocp-console/ocp-bridge-service-route.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: cluster-aas-operator-repo-bridge-service-route
+  namespace: cluster-aas-operator
+  labels:
+    caas-repo-bridge: 'true'
+    operators.coreos.com/cluster-aas-operator.cluster-aas-operator: ''
+  annotations:
+    openshift.io/host.generated: 'true'
+spec:
+  host: >-
+    claas-helm-repo-cluster-aas-operator.apps.rawagner3.cyty.p1.openshiftapps.com
```

Comment:
this will be unique for every cluster
maye we can get the `rawagner3.cyty.p1.openshiftapps.com` dynamically ?

Reply:
Done https://github.com/stolostron/cluster-templates-console-plugin/pull/17